raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/phase/IncludeResolver.java

```diff
@@ -34,10 +39,11 @@
 import org.raml.v2.internal.utils.StreamUtils;
 
 
-public class IncludeResolver implements Transformer
+public class IncludeResolver implements Transformer, ResourceURICallback
```

Comment:
though java uses `URI` all uppercase, for naming consistency with our existing classes like `UriTemplate` we should use `Uri` capitalized everywhere instead of `URI` all uppercase